game-core/src/main/java/games/strategy/engine/data/GameMap.java

```diff
@@ -127,19 +137,41 @@ public Territory getTerritory(final String s) {
   /**
    * Returns all neighbors within a certain distance of the starting territory that match the
    * condition. Does NOT include the original/starting territory in the returned Set.
+   *
+   * @param territory starting territory
+   * @param distance distance to check
```

Comment:
This javadoc perhaps could be better. What kind of check are we doing? What is the signifcance of the check?

Suggestion: reword to something like:  "distance All returned territories will be within the max distance from the starting territory. 0 represents the starting territory and no adjacencies, 1 is all immediately adjacent territories."